aldryn_client/localdev/main.py

```diff
@@ -197,7 +196,8 @@ def download_media(client, path=None):
         with tarfile.open(fileobj=media_fobj, mode='r:gz') as media_archive:
             media_archive.extractall(path=path)
 
-        click.secho('Downloaded media files into {}'.format(path), fg='green')
+    os.remove(backup_path)
```

Comment:
funny ... I was always using `os.unlink`. Apparently those functions are doing the same.